components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/IEngineConfiguration.cs

```diff
@@ -31,5 +31,12 @@ public interface IEngineConfiguration {
         /// Diagnostics interval
         /// </summary>
         TimeSpan? DiagnosticsInterval { get; }
+
+        /// <summary>
+        /// Define the maximum number of messages, 
```

Comment:
Please mention that this is the number of messages in outgress **buffer**. Without it the summary is not clear.

BTW, 200 * 256 KB is exactly 50 MB.

Reply:
updated XML comment and fixed the memory site